compilers/concrete-compiler/compiler/tests/end_to_end_tests/end_to_end_jit_test.cc

```diff
@@ -460,3 +460,18 @@ func.func @main(%arg0: !FHE.eint<3>) -> !FHE.eint<3> {
 )XXX");
   assert(err.has_value());
 }
+
+TEST(CompileMultiOutputDoubleReturn, double_return) {
```

Comment:
why not in yaml test, we try to put all the test in yaml files

Reply:
Yes sure.